### ISSUE TITLE
Integers in-between (reverse)

### ISSUE THREAD

**Issue by ShaneBeee** [SkriptLang/Skript#1481]
So checking numbers between a low and high value (between 0 and 2) works
Checking numbers between high and low value (between 2 and 0) do not work

#### Example: #
```
command /test:
	trigger:
		if 1 is between 0 and 2:
			send "1 is between 0 and 2"
		if 1 is between 2 and 0:
			send "1 is between 2 and 0"
```
The first part (between 0 and 2) sends the message
The second part (between 2 and 0) does not

**Comment by iOshawott** [comment]
the same situation when we are talking about negative numbers, but now will work only
"-1 is between -2 and 0"

and it's completely logical and fine for me because the correct number order is -2,-1,0,1,2,3
and when we are reading there's no 1 between 2 and 0 because after 2 there isn't 0

**Comment by RoboBlawck** [comment]
I have been working on comparison functions for 2 hours...now I realize there were already there in the documentation...

**Comment by Nicofisi** [comment]
I mean, I can fix this but I need someone to confirm that this is supposed to work lol

**Comment by ShaneBeee** [comment]
@Snow-Pyon  Felt that it should be fixed.... so I guess that is your confirmation ;)

**Comment by Nicofisi** [comment]
Sure then, right after @bensku merges one of the 'x times' PRs

**Comment by ShaneBeee** [comment]
Sure sure... no rush... id rather see a working version of 1.13 skript vs having a tiny little problem like this solved.... id say priority: lowest

**Comment by Nicofisi** [comment]
Priority low since I'm interested in fixing this myself, when a dev is assigned they are free to choose the priority as long as it makes any sense, after all they are supposed to take care of the thing themselves :) and also this should take only a moment. But you're right here in general

**Comment by ShaneBeee** [comment]
Well thank you for fixing it ... I do appreciate it :) 👍 for nico

**Comment by Snow-Pyon** [comment]
Negative numbers should work with this condition, since the check supports it.

**Comment by Romitou** [comment]
I think this issue has been resolved in #3248.

**Comment by APickledWalrus** [comment]
> 
> 
> This issue has been resolved in #3248.

Yep. Missed then when I opened that PR.
